spring-data-jdbc/src/test/java/org/springframework/data/jdbc/mybatis/MyBatisDataAccessStrategyUnitTests.java

```diff
@@ -15,14 +15,9 @@
  */
 package org.springframework.data.jdbc.mybatis;
 
-import static java.util.Arrays.*;
```

Comment:
Please avoid reformatting the code. 
See https://github.com/spring-projects/spring-data-build/tree/main/etc/ide for how to setup the formatter of your IDE.